pyproject.toml

```
@@ -145,7 +143,7 @@ description = "Combine coverage outputs"
 skip_install = true
 dependency_groups = ["coverage"]
 commands = [["coverage", "combine"]]
-depends = ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
+depends = ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

Looks like this needs an update.

```suggestion
depends = ["3.9", "3.10", "3.11", "3.12", "3.13", "3.14"]
```

---

Good catch; thanks!